examples/src/main/java/io/kubernetes/client/examples/InformerExample.java

```diff
@@ -11,11 +11,20 @@
 import io.kubernetes.client.models.V1ObjectMeta;
 import io.kubernetes.client.util.CallGeneratorParams;
 import io.kubernetes.client.util.Config;
+import java.util.concurrent.TimeUnit;
 
+/**
+ * A simple example of how to use the Java API
+ *
+ * <p>Easiest way to run this: mvn exec:java
+ * -Dexec.mainClass="io.kubernetes.client.examples.InformerExample"
+ *
+ * <p>From inside $REPO_DIR/examples
+ */
 public class InformerExample {
   public static void main(String[] args) throws Exception {
-
     ApiClient client = Config.defaultClient();
+    client.getHttpClient().setReadTimeout(60, TimeUnit.SECONDS);
```

Comment:
it's reasonable to apply a timeout to the list calls, but the watches normally span 10-ish minutes from informer which is jitterr'd server-side. basically expecting an infinite client-side timeout is fine and that's the client works in k/client-go.

Reply:
Yeah, but IMHO if we set an infinite client-side timeout, it might lead a dead connection and causes thread pool gets exhausted because we don't know whether the server is alive. How client-go resolve that?

Also, you'are right, 60 seconds is too short, I use this setting because WatchExample uses it :P

Reply:
```suggestion
    client.getHttpClient().setReadTimeout(0); // infinite timeout
```
oops.. am afraid the timeout settings in the watch example is misleading then from #155. that makes the watch runners shutdown in a minute-ish timeout and leads to frequent re-list which can potentially overloads the server. we should also explicitly set the timeout to `0` in the watch example. @brendandburns wdyt?

Reply:
> How does client-go resolve this?

no timeout in client-side

Reply:
OK, changed it, thanks~